be/src/storage/lake/gc.cpp

```diff
@@ -105,6 +105,19 @@ static Status list_tablet_metadata(const std::string& metadir, std::set<std::str
     });
 }
 
+static Status delete_rowset_files(FileSystem* fs, std::string_view data_dir, const RowsetMetadataPB& rowset) {
+    for (const auto& segment : rowset.segments()) {
+        auto seg_path = join_path(data_dir, segment);
+        if (auto st = ignore_not_found(fs->delete_file(seg_path)); st.ok()) {
+            LOG(INFO) << "Deleted " << seg_path;
```

Comment:
How about deleting this log to reduce log?

Reply:
How about add a configuration to control whether print this log or not? IMHO, file deletion is a quite dangerous operation and once unexpected file deletion happed, it's hard to reason about it without the delete log.